.github/workflows/test.yml

```diff
@@ -8,3 +8,9 @@ jobs:
         uses: actions/checkout@v4
       - name: test
         run: docker build .
```

Comment:
You can also kill this run and delete the Dockerfile. The current test behavior relies on this Docker setup, after integrating pytest, this setup is obsolete.